ts/Series/Treegraph/TreegraphSeriesDefaults.ts

```diff
@@ -228,6 +228,7 @@ const TreegraphSeriesDefaults = {
      */
     dataLabels: {
         defer: true,
+        allowOverlap: false,
```

Comment:
I don't think we need this line to add, as Treegraph series is inheriting this option and API states correctly as `false`: https://api.highcharts.com/highcharts/series.treegraph.dataLabels.allowOverlap

Reply:
For some reason I struggled with reffering to it but I will check again with your suggestion!

Reply:
Your suggestion works, my bad!